Meta/Keywords/scripting_library/data/writing/plotting.py

```diff
@@ -149,7 +149,7 @@ async def plot(ctx, title, x=None,
         elif isinstance(z, list) and not isinstance(x, list):
             x = [x] * len(z)
         if len(x) and \
-                not ctx.symbol_writer.contains_row(title,
+                not await ctx.symbol_writer.contains_row(title,
```

Comment:
👍 
Can you also add it in like 185 please ?

Reply:
I added it